etc/kayobe/kolla/config/neutron/l3_agent.ini

```diff
@@ -0,0 +1,3 @@
+interface_driver = multi
```

Comment:
Same as above - do these need to be in config section?

Reply:
Hmm if it works without this config then maybe its not even required?